### PR TITLE
Create percyjackson.ms

### PR DESCRIPTION
**Pull request by SirRicardo**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buff/percyjackson.ms [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                "/jump 0-3",
+                "/run 0-4",
+                "/carryplayer",
+                "/dismountplayer",
+                "/haste 0-5"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 12 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 5 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buff/percyjackson.ms]: this is a weird number of hearts

File: CommandHelper/LocalPackages/commands/buff/percyjackson.ms [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                "/shot give ". @plr ." watershot",
+                "/give ". @plr ." minecraft:diamond_sword 1 19:16 62:5 name:&3R&9i&3p&9t&3i&9d&e",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buff/percyjackson.ms]: why so much knockback?
lure doesnt work on swords

File: CommandHelper/LocalPackages/commands/buff/percyjackson.ms [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                "/carryplayer",
+                "/dismountplayer",
+                "/haste 0-5"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 12 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 4 true",
+                "/shot give ". @plr ." watershot",
+                "/give ". @plr ." minecraft:diamond_sword 1 19:2 10:2 name:&3R&9i&3p&9t&3i&9d&e",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buff/percyjackson.ms]: why a curse of binding?

Comment (SirRicardo) [inline CommandHelper/LocalPackages/commands/buff/percyjackson.ms]: so others cant get the sword

Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buff/percyjackson.ms]: its already like that

**Pull request merged**